src/overlays/actors/ovl_Bg_Spot05_Soko/z_bg_spot05_soko.c

```diff
@@ -21,17 +32,77 @@ const ActorInit Bg_Spot05_Soko_InitVars = {
     (ActorFunc)BgSpot05Soko_Update,
     (ActorFunc)BgSpot05Soko_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot05_Soko/BgSpot05Soko_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot05_Soko/BgSpot05Soko_Destroy.s")
+static InitChainEntry initChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
+};
+
+static Gfx* dListTbl[] = {
+    0x06000840,
+    0x06001190,
+};
+
+static void BgSpot05Soko_Init(BgSpot05Soko* this, GlobalContext* globalCtx) {
+
+    Actor* thisx = &this->dyna.actor;
+    u32 sp24[2];
+    sp24[0] = 0;
```

Comment:
I don't think this is supposed to be an array.

Reply:
If it's not an array, should I I just make it padding?

Reply:
If that's required for stack allocation, yes.

Reply:
I tried a lot of different things and nothings matching if I don't make sp24 an array

Reply:
The stack isnt even changing, even if I make the pad 5 words long. It keeps going to -0x20. But when i make sp24 an array, it matches with stack at -0x30

Reply:
Oh I think I found the issue